scripts/verify_tasks.py

```
@@ -0,0 +1,204 @@
+                    frontmatter[key.strip()] = value.strip()
+
+        # Extract acceptance criteria
+        ac_match = re.search(r'<!-- AC:BEGIN -->(.*?)<!-- AC:END -->', content, re.DOTALL)
+        ac_items = []
+        if ac_match:
+            ac_content = ac_match.group(1)
+            # Find checked items
+            checked_items = re.findall(r'- \[x\]\s*#(\d+)\s*(.+)', ac_content)
+            unchecked_items = re.findall(r'- \[ \]\s*#(\d+)\s*(.+)', ac_content)
+            ac_items = [(int(num), text.strip(), True) for num, text in checked_items] + \
+                      [(int(num), text.strip(), False) for num, text in unchecked_items]
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)